code/controlconfig/controlsconfig.h

```diff
@@ -361,6 +372,16 @@ class CC_bind {
 	 */
 	bool empty() const;
 
+	/**
+	 * Takes the given binding.
+	 */
+	void take(CID _cid, short _btn, char _flags = 0);
+
+	/**
+	 * Validates the binding, clearing it if necassary
+	 */
+	void validate();
+
```

Comment:
Just a question.  Why was this moved?

Reply:
Moved for organization, to better group the Getter methods together.